src/main/kotlin/app/extractors/CsharpExtractor.kt

```diff
@@ -0,0 +1,38 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Liubov Yaronskaya (lyaronskaya@sourcerer.io)
+
+package app.extractors
+
+import app.model.CommitStats
+import app.model.DiffFile
+import java.io.File
+
+class CsharpExtractor : ExtractorInterface {
+    override fun extract(files: List<DiffFile>): List<CommitStats> {
+        TODO("not implemented")
+    }
+
+    override fun extractImports(fileContent: List<String>): List<String> {
+        val libraries = mutableSetOf<String>()
+
+        val csLibraries = File("data/libraries/cs_libraries.txt")
```

Comment:
This should be definitely optimized cuz everytime extractImports called a file will be read. IO operations take a lot of time. I'll do it.

Reply:
.